sources/settings-extensions/ntp/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "settings-extension-ntp"
+version = "0.1.0"
+authors = ["Sam Berning <bernings@amazon.com>"]
+license = "Apache-2.0 OR MIT"
+edition = "2021"
+publish = false
+
+[dependencies]
```

Comment:
The `settings-sdk` emits logs to `stderr`, and I realized after talking with @zmrow that I never initialized the logger in the `motd` extension.

I would take a dependency on [`env_logger`](https://crates.io/crates/env_logger) and initialize the logger just before initializing the extension with `env_logger::init()`.